jac/jaclang/plugin/default.py

```diff
@@ -561,6 +562,29 @@ def get_context() -> ExecutionContext:
         """Get current execution context."""
         return ExecutionContext.get()
 
+    @staticmethod
+    @hookimpl
+    def purge_graph(root: Optional[Root] = None) -> int:
```

Comment:
I'd just call this `reset_graph`, `purge` sounds too intense :)

Reply:
agreed haha